common/templates/components/read-only/template.njk

```diff
@@ -2,7 +2,7 @@
 
 {% if params.title %}
 <div class="upw-read-only__header">
-    <h2 class="govuk-heading-m upw-read-only__heading">{{ params.title }}</h2>
+    <h3 class="govuk-heading-m upw-read-only__heading">{{ params.title }}</h2>
```

Comment:
Not related to this ticket, but fixes an issue where we're having table captions at the same level as section headings